README.md

```diff
@@ -32,11 +32,15 @@ This is a basic everyday Calculator with following features:
 
 
 # Run Locally
+- From the 'Code' section of your forked repo, copy the URL of your forked repo under HTTPS/ SSH/ Github CLI.
+- Open Git Bash terminal.
+- Change the current working directory to the location where you want the cloned directory.
+- Type ```git clone```, and then paste the URL you copied earlier. It will look like this, with your GitHub username instead of ```YOUR-USERNAME```:
 
-- Clone your forked repository
```

Comment:
Don't remove this topic @Arcturus22

Reply:
Btw the steps I have written mention it.

Reply:
Is the line Clone your forked repository only needed?
Should I remove the detailed steps and only mention that line?

Reply:
No need of them, This is required so don't remove it ok? @Arcturus22

Reply:
> Is the line Clone your forked repository only needed? Should I remove the detailed steps and only mention that line?

Yes @Arcturus22

Reply:
@YoshithaRathnayake 
Please verify this.
![Img-0](https://user-images.githubusercontent.com/99889376/197382097-51b7a14d-be04-4b66-9def-2c636faef01a.jpg)

Reply:
No need of that description @Arcturus22

Reply:
No need of that description @Arcturus22 remove that